src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -189,23 +189,48 @@ def compute_contact_forces(
         model: js.model.JaxSimModel,
         data: js.data.JaxSimModelData,
         link_forces: jtp.MatrixLike | None = None,
+        joint_force_references: jtp.VectorLike | None = None,
     ) -> tuple[jtp.Vector, tuple[Any, ...]]:
+        """
+        Compute the contact forces.
+
+        Args:
+            position: The position of the collidable point.
+            velocity: The linear velocity of the collidable point.
+            model: The `JaxSimModel` instance.
+            data: The `JaxSimModelData` instance.
+            link_forces:
+                Optional `(n_links, 6)` matrix of external forces acting on the links,
+                expressed in the same representation of data.
+            joint_force_references:
+                Optional `(n_joints,)` vector of joint forces.
+
+        Returns:
+            A tuple containing the contact forces.
+        """
 
         link_forces = (
             link_forces
             if link_forces is not None
             else jnp.zeros((model.number_of_links(), 6))
         )
 
+        joint_force_references = (
+            joint_force_references
+            if joint_force_references is not None
+            else jnp.zeros((model.number_of_joints(),))
```

Comment:
```suggestion
            else jnp.zeros(model.number_of_joints())
```

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/233/commits/64bef4dfb446ff8103d67924f6cee23da010896e